main.tex

```diff
@@ -28,6 +28,15 @@
 \setcounter{secnumdepth}{3}
 \setcounter{tocdepth}{3}
 
+% Added by Dominic Lunde (4/2019) to avoid windows and orphans
```

Comment:
Not that I don't want to give you credit, but the author will appear in the diff, no reason to call out the author here in line.